internal/tofu/node_resource_abstract_test.go

```diff
@@ -230,7 +234,8 @@ func TestNodeAbstractResource_ReadResourceInstanceState(t *testing.T) {
 
 			ctx.ProviderProvider = providers.Interface(mockProvider)
 
-			got, readDiags := test.Node.readResourceInstanceState(ctx, test.Node.NodeAbstractResource.Addr.Resource.Instance(addrs.NoKey).Absolute(addrs.RootModuleInstance))
+			//TODO same change as line 220, please double check
```

Comment:
Those addr changes look good to me: code under test should behave the same way, and manual addresses match the state.